content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md

```diff
@@ -510,21 +420,11 @@ $ kubectl create -n istio-system secret generic httpbin-credential --from-file=k
 
     {{< /text >}}
 
-1. Instead of creating a `httpbin-credential` secret to hold all the credentials, you can
-   create two separate secrets:
+Istio supports reading a few different formats, to support integration with various tools such as [cert-manager](/docs/ops/integrations/certmanager/):
 
-    * `httpbin-credential` holds the server's key and certificate
-    * `httpbin-credential-cacert` holds the client's CA certificate and must have the `-cacert` suffix
-
-    Create the two separate secrets with the following commands:
-
-    {{< text bash >}}
-    $ kubectl -n istio-system delete secret httpbin-credential
-    $ kubectl create -n istio-system secret generic httpbin-credential  \
-    --from-file=key=httpbin.example.com.key --from-file=cert=httpbin.example.com.crt
-    $ kubectl create -n istio-system secret generic httpbin-credential-cacert  \
-    --from-file=cacert=example.com.crt
-    {{< /text >}}
+* A Secret with keys `tls.key` and `tls.crt`, as described above. For mutual TLS, a `ca.crt` key can be used.
```

Comment:
Could you use "A TLS secret with keys ... " and A generic secret with keys ... "?

Reply:
done